Makefile

```diff
@@ -182,8 +186,9 @@ $(RELEASE_DIR):
 build-release-artifacts: build-cluster-templates build-infra-yaml build-metadata-yaml build-host-agent-binary
 
 build-cluster-templates: $(RELEASE_DIR) cluster-templates
-	cp $(BYOH_TEMPLATES)/v1beta1/cluster-template.yaml $(RELEASE_DIR)/cluster-template.yaml
-	sed -i -e 1,20d $(RELEASE_DIR)/cluster-template.yaml
```

Comment:
Question: Will keeping the `CRS/CM` in the cluster template is going to have any impact? Do we need to test it?

Reply:
We are not keeping it in the vm and docker templates.